### PR TITLE
Bump doctrine deps

### PR DESCRIPTION
**Pull request by rullzer**

### CODE REVIEW

File: doctrine/common/composer.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.6.x-dev"
+            "dev-master": "2.5.x-dev"
```
Comment (LukasReschke) [inline doctrine/common/composer.json]: That looks fishy? From 2.6 to 2.5?

Comment (DeepDiver1975) [inline doctrine/common/composer.json]: if I get this right - it's unrelated to us.

Comment (rullzer) [inline doctrine/common/composer.json]: mmm right. let me actaully add the composer stuff to the gitignore...

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**